readthedocs/proxito/views/mixins.py

```diff
@@ -336,77 +335,6 @@ def system_redirect(
         resp["X-RTD-Redirect"] = RedirectType.system.name
         return resp
 
-    def canonical_redirect(
```

Comment:
Most of these don't need to be in a mixin class, they are just a collection of functions, this is they don't change the class that inherits from them.

Reply:
You didn't change this logic, right? Just moving it around?

Reply:
yeah, just moving